src/homepage/howTo/deleted-message-vault.html

```diff
@@ -0,0 +1,300 @@
+    KIND, either express or implied. See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+<link href="assets/css/lightbox.css" rel="stylesheet">
+<link href="assets/css/lity.min.css" rel="stylesheet" />
+<div id="wrapper">
+    <div class="apache_ref">
+        <a href="https://www.apache.org" alt="apache foundation link"><img src="https://www.apache.org/foundation/press/kit/asf_logo.svg" title="apache foundation logo"/></a>
+    </div>
+    <div class="apache_ref_mobile">
+        <a href="https://www.apache.org" alt="apache foundation link">The Apache Software Foundation</a>
+    </div>
+    <div class="apache_ref_left">
+        <a href="https://www.apache.org/events/current-event.html" alt="apache foundation event"><img src="https://www.apache.org/events/current-event-234x60.png" title="apache foundation event logo"/></a>
+    </div>
+    <div class="apache_ref_left_mobile">
+        <a href="https://www.apache.org/events/current-event.html" alt="apache foundation event"><img src="https://www.apache.org/events/current-event-234x60.png" title="apache foundation event logo"/></a>
+    </div>
+
+    <!-- Header -->
+    <header id="header" class="alt">
+        <div class="logo"><a href="/index.html" alt="Apache James"><img src="/images/james.svg" alt="james logo"/></a></div>
+        <h1 class="hidden">James Enterprise Mail Server</h1>
+        <h2>Emails at the heart of your business logic</h2>
+    </header>
+
+    <!-- Main -->
+    <div id="main">
+
+        <!-- Introduction -->
+        <section id="intro" class="main special">
+            <div class="">
+                <div class="content align-left">
+                    <header class="major">
+                        <h1><b>Deleted Message Vault</b></h1>
+                    </header>
+
+                    <p>
+                        This document will help you understand what is Deleted Message Vault and what does it for. And finally,
+                        present to you, how to setup James Server with this feature.
+                    </p>
+
+                    <ul>
+                        <li>What is Deleted Message Vault</li>
+                        <li>Make Deleted Message Vault available in James</li>
+                        <li>Make James uses Deleted Message Vault by configuring Pre Deletion Hooks</li>
+                        <li>Start a James Server with Deleted Message Vault</li>
+                        <li>WebAdmin capability on top of Deleted Message Vault</li>
+                    </ul>
+
+                    <header class="major">
+                        <h2><b>What is Deleted Message Vault</b></h2>
+                    </header>
+
+                    <p>Deleted Message Vault is a plugin in James, it allows you to:</p>
+                    <ul>
+                        <li>store deleted messages from users</li>
+                        <li>search for deleted messages by various criteria</li>
```

Comment:
We never search. We restore and we export.

Reply:
I'm talking about the DeletedMessageVault API, not the web admin